### DISCUSSION TITLE
Crash when building with LLVM-clang

### DISCUSSION THREAD

**Discussion by gut5** [helix-editor/helix#9167] [Troubleshooting]
### Summary

Crashes when attempting to build grammars using llvm-clang (distro is openSUSE Tumbleweed in WSL2).

```
  Failure 169/171: templ Failed to execute C/C++ compiler
  Failure 170/171: log Failed to execute C/C++ compiler
  Failure 171/171: dbml Failed to execute C/C++ compiler

  --- stderr
  thread 'main' panicked at 'Failed to compile tree-sitter grammars: 171 grammars failed to build', helix-term/build.rs:7:14
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: failed to compile `helix-term v23.10.0 (/home/guts/helix/helix-term)`, intermediate artifacts can be found at `/home/guts/helix/target`
```

### Reproduction Steps

1. install llvm-clang
2. use build steps 
```
git clone https://github.com/helix-editor/helix
cd helix
cargo install --path helix-term --locked
```

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux - opensuse tumbleweed WSL2

### Terminal Emulator

Windows Terminal

### Installation Method

source

### Helix Version

23.10

**Comment by pascalkuthe** [comment]
Is the `cc` Co.mand available on your computer? What is your `CC` environment variable? Helix can not find a c compiler

**Reply by gut5** [reply]
The CC environment variable is empty even after installing gcc-c++ in opensuse. Building with gcc-c++ does work however. It's just when I use llvm-clang to build (in a WSL2 environment).
